.github/workflows/build.yml

```diff
@@ -128,3 +128,37 @@ jobs:
           set -eu -o pipefail
           . .github/workflows/scripts/buildkitenvs/${{ matrix.buildkit_mode}}
           go test -v ./integration --addr="${COPA_BUILDKIT_ADDR}" --copa="$(pwd)/copa" -timeout 0
+
+  test-plugin:
+    needs: test-patch
```

Comment:
why do we need to wait for `test-patch`? would `needs: build` work?

Reply:
yes, added